src/GraphQL.Tests/Types/DateGraphTypeTests.cs

```diff
@@ -20,52 +20,12 @@ public void serialize_string_to_date()
             });
         }
 
-        [Fact]
-        public void serialize_local_date_returns_date_only()
```

Comment:
Don't you think that we should to keep these tests, just now expect an exception from them about time component?